src/ca/wolfram/beta/symath/ConstantNode.java

```diff
@@ -0,0 +1,44 @@
+package ca.wolfram.beta.symath;
+
+import java.util.List;
+
+/**
+ * Created by Allan Wang on 2017-05-06.
+ */
+public class ConstantNode implements MathNode {
+
+    private final double constant;
+
+    private ConstantNode(double constant) {
+        this.constant = constant;
+    }
+
+    public static ConstantNode create(double constant) {
```

Comment:
I'll turn these into normal constructors since they are now package private and initiated through BaseNode statically